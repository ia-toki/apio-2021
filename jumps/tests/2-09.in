209636825ecd2c38299498cd26890ef71dd128c2
200 200
102 184 154 158 108 14 1 144 135 181 44 51 159 10 94 138 83 150 76 112 63 134 65 8 170 127 182 117 100 55 19 9 61 180 28 80 111 165 50 171 101 167 37 90 157 20 162 43 24 6 125 146 188 92 97 166 128 35 115 110 106 116 141 120 30 18 147 195 54 175 196 186 189 99 47 114 96 31 148 59 160 145 42 168 57 62 15 109 41 27 46 163 60 129 13 16 187 133 183 64 118 34 79 123 39 72 5 137 21 113 22 48 174 52 66 121 2 136 179 142 40 152 86 23 197 70 71 67 12 17 124 198 149 81 87 191 45 185 140 89 91 169 49 161 105 95 84 38 26 139 78 156 190 194 75 25 131 172 103 93 3 130 53 82 88 58 33 122 132 32 56 74 107 193 126 4 151 104 119 178 199 36 176 68 200 77 98 192 11 69 155 73 177 85 153 29 143 164 7 173
0 0 199 199
52 73 87 135
60 69 146 171
144 152 157 193
102 119 170 190
94 97 171 179
101 146 189 197
25 77 113 154
88 100 119 140
7 7 80 90
17 56 64 66
146 168 185 196
111 174 185 194
57 65 103 199
22 89 103 153
16 17 175 183
112 198 199 199
94 114 166 193
22 160 180 199
41 57 123 136
0 13 157 165
1 14 127 175
4 64 131 178
109 156 191 199
18 23 37 174
66 70 113 194
95 104 130 148
4 7 72 140
16 58 97 183
167 183 193 193
12 14 57 110
78 81 133 188
75 112 183 187
56 82 189 191
0 87 99 114
50 81 137 165
56 131 138 146
68 69 144 184
71 74 91 145
137 158 189 197
29 58 146 182
131 186 197 197
7 11 144 148
78 89 97 188
33 137 143 146
49 67 177 197
96 119 163 173
9 17 35 45
17 146 187 196
7 22 129 184
4 74 197 197
6 81 83 126
77 87 154 159
51 126 149 187
15 24 124 132
26 75 112 117
61 77 182 193
17 60 178 194
26 158 196 197
76 98 133 150
89 169 180 196
39 50 133 139
107 150 152 183
124 156 187 189
119 185 187 198
70 160 161 194
88 143 147 153
25 62 155 162
15 33 48 118
58 119 138 174
63 106 165 182
104 152 165 173
16 20 191 198
37 101 154 161
29 101 143 173
85 188 196 199
108 125 182 187
36 105 165 191
176 182 188 197
53 80 102 155
64 144 154 183
26 103 140 140
4 72 165 191
12 17 108 108
21 145 155 172
36 56 153 186
26 49 124 128
83 126 182 195
53 130 153 188
20 87 89 191
1 32 136 158
26 144 152 176
12 14 31 153
10 65 76 114
140 172 196 197
64 156 157 166
123 182 186 197
76 99 136 176
120 184 186 193
33 63 123 161
18 52 171 176
84 136 142 167
17 37 82 125
124 186 188 196
2 42 145 174
120 146 180 189
62 70 195 195
53 138 147 194
2 10 150 155
30 33 88 138
37 171 193 194
101 126 160 164
13 46 68 115
41 51 67 104
62 67 144 186
193 198 199 199
151 197 198 199
95 192 196 197
102 184 199 199
71 83 183 191
11 35 128 138
20 124 161 176
148 155 190 195
21 38 193 196
30 34 183 184
15 163 195 196
45 88 101 147
1 61 146 153
55 62 190 197
79 81 91 152
38 77 134 198
64 162 170 198
4 38 125 181
24 43 44 197
62 193 195 196
45 45 47 91
1 3 98 174
82 89 134 150
52 122 177 188
4 24 82 98
104 133 169 192
71 181 194 195
34 166 189 195
146 187 195 196
83 191 195 197
47 179 185 186
113 186 196 197
20 36 83 184
86 193 195 198
8 9 49 52
42 65 107 139
162 163 165 196
3 90 136 144
114 156 196 198
95 105 160 182
107 165 167 184
89 141 159 193
1 87 99 103
118 141 180 199
6 122 145 194
87 146 196 196
4 6 177 196
0 29 44 130
6 30 168 172
17 17 56 146
86 137 196 197
3 20 44 96
63 114 161 163
24 74 190 197
17 83 165 166
21 31 140 156
21 35 38 96
107 147 169 179
11 35 98 164
19 175 199 199
129 191 193 193
1 3 41 100
1 90 173 199
88 145 181 199
57 83 103 193
8 80 122 145
15 59 180 193
5 8 134 199
108 197 199 199
24 134 144 144
33 35 109 152
17 65 180 181
10 12 33 114
16 100 176 182
64 113 120 179
21 60 120 139
69 93 134 150
158 172 173 187
26 65 180 194
36 59 117 194
21 140 146 160
68 91 104 194
29 37 125 155
38 65 72 103
22 176 189 194
