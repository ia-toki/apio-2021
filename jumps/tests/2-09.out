fc5005cd102fffc504bc1615010ebf88823d65f3
OK
-1
1
-1
2
3
-1
-1
1
1
-1
2
-1
-1
4
1
8
1
3
1
3
-1
5
4
-1
3
1
2
6
3
-1
4
4
4
-1
-1
-1
-1
4
1
-1
5
-1
-1
2
-1
4
-1
-1
-1
6
-1
1
-1
2
5
-1
4
5
-1
-1
1
-1
1
-1
2
1
1
-1
1
-1
3
-1
-1
-1
-1
-1
3
3
-1
1
1
-1
3
-1
-1
5
4
3
2
1
-1
1
3
-1
-1
1
-1
-1
-1
3
-1
1
4
-1
-1
1
-1
1
-1
4
-1
-1
3
2
4
1
1
-1
-1
4
5
-1
-1
-1
7
-1
1
-1
-1
2
3
1
5
1
-1
2
4
-1
3
-1
1
-1
-1
-1
-1
-1
-1
4
2
2
3
2
-1
-1
3
1
1
-1
1
3
-1
8
1
-1
4
-1
2
-1
-1
-1
-1
1
-1
4
-1
-1
1
3
2
1
1
5
8
1
-1
4
5
3
3
1
3
-1
1
5
3
1
1
5
-1
-1
